### PR TITLE
Updated the 'changing content source' document

### PR DESCRIPTION
**Pull request by AkshayGadhaveRH**
Updated the changing content source document with information about single host.

Cherry-pick into:

* [ ] Foreman 3.3
* [ ] Foreman 3.2
* [x] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

**Review by melcorr — COMMENTED** [review]
In my opinion, this update is regressive.

**Comment by melcorr** [comment]
Does " . Select the check box of one or more hosts whose content source you want to change. " not cover it?

**Comment by AkshayGadhaveRH** [comment]
Hey @melcorr the intention is to inform that the "change content source" option is also available in the "Schedule a job" list present on the host's detail page. 
Maybe I can reword it to "Optionally, for a single host, you can change the content source from *Hosts* > *All Hosts* >  Select the host > *Schedule a job* > select *Change Content Source*". Or does this sound too confusing?

**Comment by AkshayGadhaveRH** [comment]
Hi @melcorr thanks, I agree with you. I can't think of any other section to place this. Should I close this PR?
